tests/conftest.py

```diff
@@ -58,6 +89,7 @@ def manim_caplog(caplog):
 @pytest.fixture
 def config():
     saved = manim.config.copy()
+    manim.config.renderer = "cairo"
```

Comment:
Should this be reverted?

Reply:
This fixed a bunch of tests that were mistakenly run with the opengl renderer, we should keep it for now.

Curiously, the pipeline failed again in the pathological case, am looking into details.